src/screens/EventDetailsScreen/EventOverview.js

```diff
@@ -63,32 +61,30 @@ const EventOverview = ({ event }: Props) => {
         onPress={() => showLocation(...eventLocation)}
         source={locationIcon}
       >
-        <TextLink>{event.fields.locationName[locale]}</TextLink>
+        <TextLink>{event.fields.locationName}</TextLink>
         {event.fields.addressLine1 && (
-          <Text type="small">{event.fields.addressLine1[locale]}</Text>
+          <Text type="small">{event.fields.addressLine1}</Text>
         )}
         {event.fields.addressLine2 && (
-          <Text type="small">{event.fields.addressLine2[locale]}</Text>
-        )}
-        {event.fields.city && (
-          <Text type="small">{event.fields.city[locale]}</Text>
+          <Text type="small">{event.fields.addressLine2}</Text>
         )}
+        {event.fields.city && <Text type="small">{event.fields.city}</Text>}
         {event.fields.postcode && (
-          <Text type="small">{event.fields.postcode[locale]}</Text>
+          <Text type="small">{event.fields.postcode}</Text>
         )}
       </IconItem>
 
       <IconItem source={ticketsIcon}>
         <Text type="h4" color="lightNavyBlueColor">
           {formatLongEventPrice(
-            event.fields.eventPriceLow[locale],
-            event.fields.eventPriceHigh[locale]
+            event.fields.eventPriceLow,
+            event.fields.eventPriceHigh
           )}
         </Text>
       </IconItem>
 
       {event.fields.venueDetails &&
```

Comment:
In the new event type `event.fields.venueDetails` is always an array, so we should be able to remove this check. Similar to `accessibilityOptions`.